CONTRIBUTING.md

```diff
@@ -203,6 +203,20 @@ cd packages/opentelemetry-module-name
 npm run watch
 ```
 
+#### TypeScript version
+
+TypeScript version used to compile the pacakges is `v5.6.3`. If you plan to make your own instrumentation script
+in a `.ts` file it is recommended to use same version or higher.
+
+<!-- TODO: review the update policy -->
+Also TypeScript is meant to be updated in compatible verisons of `5.x`. However there could be scenarios
+where we might don't want to update the minor version:
```

Comment:
This is likely my TypeScript-ignorance: What is TypeScript's definition of "compatible versions", if not all of "5.x"? 

Or could I be misunderstanding what you are saying? Are you saying "all updates of TS 5.x to a later 5.x version is meant to be compatible, but here is a list of reasons that might not be true for us: ..."?